app/src/components/percentage-breakdown-input.tsx

```diff
@@ -0,0 +1,226 @@
+"use client";
+
+import { ChevronDownIcon, ChevronUpIcon } from "@chakra-ui/icons";
+import {
+  FormControl,
+  FormLabel,
+  HStack,
+  Icon,
+  Input,
+  InputGroup,
+  InputRightAddon,
+  InputRightElement,
+  Popover,
+  PopoverArrow,
+  PopoverBody,
+  PopoverContent,
+  PopoverTrigger,
+  Text,
+} from "@chakra-ui/react";
+import React, { FC } from "react";
+import { FieldError } from "react-hook-form";
+import {
+  FoodIcon,
+  GardenIcon,
+  IndustrialIcon,
+  PaperIcon,
+  TextilesIcon,
+  WoodIcon,
+} from "./icons";
+import type { TFunction } from "i18next";
+
+const breakdownCategories = [
```

Comment:
I think it would be a good idea not to hard code this and make it more dynamic, so this percentage breakdown component can be reused easily

Reply:
Added a TODO comment. We will do this once there is a use case for multiple different kinds of percentage breakdown inputs.